docs/source/users/memory-game.rst

```diff
@@ -0,0 +1,61 @@
+Memory game
+-----------
+
+For the purpose of not disclosing to participants that language attitudes are being measured,
+a memory game was implemented to act as a distractor after the completion of linguistic tasks
+such as the LSBQe and AToL.  
+
+.. figure:: figures-mt/memory_task_homescreen.png
+    :name: memory_task_homescreen
+    :width: 600
+    :alt: Screenshot of Research Client homescreen 
+
+    Research Client homescreen 
+
+The memory game consists of 20 picture cards in which participants are tasked to match each card
+with its respective pair in the shortest amount of time possible. Once a card has been clicked,
+only then the timer will begin. As seen in :numref:`memory_task_1`, all cards are faced down and
+only two cards are able to be revealed at a time, therefore the participant must try to memorise
+the card locations. 
+
+.. figure:: figures-mt/memory_task_1.png
+    :name: memory_task_1
+    :width: 600
+    :alt: Screenshot of memory game 
+
+    Starting screen of memory game 
+
+.. figure:: figures-mt/memory_task_matching_pairs.png
+    :name: memory_task_matching_pairs
+    :width: 600
+    :alt: Screenshot of matching identical cards in the memory task
+
+    Matching identical card pairs in the memory task
+
+There are a total of two rounds in which the participant is given the opportunity to achieve a higher score.
+
+After completing the memory task, a congratulatory results screen will appear.
+Results are displayed after each round as seen in :numref:`memory_task_first_results` and  
+data for the memory task will not be saved.
+
+However, as this is purely a distractor task, participants will be informed that their
```

Comment:
I think the 'However, ...' is unnecessary. The sentence can just start with "Participants will be informed ..."